examples/cql_collections.rs

```diff
@@ -0,0 +1,56 @@
+use anyhow::Result;
+use scylla::cql_to_rust::FromRow;
+use scylla::macros::FromRow;
+use scylla::transport::session::{IntoTypedRows, Session};
+use std::env;
+use std::collections::HashMap;
+
+#[tokio::main]
+async fn main() -> Result<()> {
+
+    // Create connection
+    let uri = env::var("SCYLLA_URI").unwrap_or("127.0.0.1:9042".to_string());
+
+    println!("Connecting to {} ...", uri);
+
+    let session = Session::connect(uri, None).await?;
+    session.refresh_topology().await?;
+
+    session.query("CREATE KEYSPACE IF NOT EXISTS ks WITH REPLICATION = {'class' : 'SimpleStrategy', 'replication_factor' : 1}", &[]).await?;
+
+    // Create the table
+    session
+        .query(
+            "CREATE TABLE IF NOT EXISTS teams (id int, team map<text, text>, primary key id)",
+            &[],
+        )
+        .await?;
+
+    // Send some data
+    // TODO after implementation
+
+    // Recieve data in singular tuple
+
+    if let Some(rows) = session.query("SELECT team FROM teams", &[]).await? {
+        for row in rows.into_typed::<(HashMap<i32, String>,)>() {
```

Comment:
Why do you cast to a hash map of i32 -> String, if the column is specified above as `map<text, text>`? Shouldn't the column instead be defined as `map<int, text>` in the database?

Reply:
Fixed.